pkgs/by-name/ed/edex-ui/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+appimageTools.wrapType2 rec {
+  pname = "edex-ui";
+  name = "eDEX-UI";
```

Comment:
```suggestion
```
name is automatically computed as `"$pname-$version"`.
Use eDEX-UI directly below.